scan_topic: /base_laser_rear/scan
transform_link: base_link
x_lower_limit: -2.0
x_upper_limit: 0.0
y_lower_limit: -1.0
y_upper_limit: 1.0